### DISCUSSION TITLE
more Stamps

### DISCUSSION THREAD

**Discussion by Ambaquista** [sumatrapdfreader/sumatrapdf#3450] [Q&A]
How to add more Stamps to the annotation function?

**Comment by GitHubRulesOK** [comment]
Currently not possible (uses the inbuilt MuPDF stock text, e.g. Adobe standard range), and be cautious SumatraPDF "may" have problems with "custom" stamps, there are a few open issues.

**Comment by Ambaquista** [comment]
OK, thanks.
Congratulations for your GOOD app!
*Carlos Alberto Carranca*
***@***.***
024 924360875
024 226421001 * 300015
